### PR TITLE
US translation strings

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
This adds US translation strings for (almost all) the languages that already had US translations.

### CODE REVIEW

File: web/public/locales/sv.json [hunk 1]
```diff
@@ -1850,6 +1857,14 @@
     },
     "ZW": {
       "zoneName": "Zimbabwe"
+    },
+
+    "US": {
+      "zoneName": "Amerikas f�renta stater"
```
Comment (madsnedergaard) [inline web/public/locales/sv.json:1863]: Is it just Github that cannot show the `ö` or is something else going on? 🤔

Comment (VIKTORVAV99) [reply web/public/locales/sv.json:1863]: Strange, I'll look into if the coding changed on the file. Or if there is something else going on.

Comment (VIKTORVAV99) [reply web/public/locales/sv.json:1863]: I think I found the issue, it's neither the coding or github. It's the translation helper...

I'll update the translations manually and then look into that script to see what's going on under the hood.

**Review by unitrium — APPROVED** [review]

**Comment by madsnedergaard** [comment]
Why did you remove all the DE zone-name translations, was that on purpose? @VIKTORVAV99

**Comment by VIKTORVAV99** [comment]
> ~~Why did you remove all the DE zone-name translations, was that on purpose? @VIKTORVAV99~~
> Nevermind, I didn't notice the problem in merge commits 😅 

Yep that was on purpose, cause I accidentally included them twice 😅

I think I removed the correct "duplicate".

**Pull request merged**